migrations/20210612_initial.sql

```diff
@@ -0,0 +1,104 @@
+CREATE TABLE profiles (
+  profile_id INTEGER NOT NULL PRIMARY KEY,
```

Comment:
do we expect `profile_id` to be sqlite's _"Integer Primary Key"_ in this case (i.e. an alias for rowid) or not? If we do, it's not clear it'd be an alias for rowid - see analogous bug described for DESC - https://sqlite.org/lang_createtable.html#rowid. If we don't either way it's ambiguous. Also `INTEGER PRIMARY KEY` primary key would already not allow nulls, unlike some other sqlite's primary keys - this would remove the ambiguity.

anyway, why not make `display_name` PK?

Reply:
> do we expect profile_id to be sqlite's "Integer Primary Key" in this case

I thought so. Not sure from comments what exactly I should do to make it unambiguous?

> I thought a profile is one of this user's profiles on this chat app

I thought to use it for the user as well, having multiple users, but then I removed users table - it's easier to simply have multiple db files (I think...). But the profile should still be stored in the db - we could have it in .ini and when there are changes compared with DB - update it and send updates to contacts... Not sure yet.